hr_expense_sequence/data/hr_expense_data.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0"?>
```

Comment:
I really see no point in removing this, even more - probably it should have been `<?xml version="1.0" encoding="UTF-8" ?>`